chain/index/ddls.go

```diff
@@ -52,37 +52,40 @@ var ddls = []string{
 	`CREATE INDEX IF NOT EXISTS idx_height ON tipset_message (height)`,
 
 	`CREATE INDEX IF NOT EXISTS event_entry_event_id ON event_entry(event_id)`,
+
+	`CREATE INDEX IF NOT EXISTS idx_tipset_key_reverted_message_id ON tipset_message (tipset_key_cid, reverted, message_id)`,
```

Comment:
@akaladarshi Let's remove this as having complex composite Indices like this has caused issues in the past for read performance. We already have the tabled indexed by `tipset_key_cid, message_id` and that should be enough.

Reply:
I just added it because in many places we are fetching based on these three columns, especially `tipset_key_cid` and `reverted`.

Anyways, I will remove it.

Reply:
Make sense. Thanks for clarifying :pray:. 

Also I have removed the index.